supervision/detection/core.py

```diff
@@ -473,6 +474,10 @@ def from_roboflow(cls, roboflow_result: dict) -> Detections:
             >>> detections = sv.Detections.from_roboflow(roboflow_result)
             ```
         """
+        if isinstance(roboflow_result, BaseModel):
```

Comment:
Like I said, I want to avoid adding `pydantic` to dependencies. Let's try to parse and [suppress](https://docs.python.org/3/library/contextlib.html#contextlib.suppress) any potential exception.